### PR TITLE
Automated Test - COMMERCE-6087 | master

### PR DESCRIPTION
**Pull request by ocesar9**

### CODE REVIEW

**Comment by c3saroliveira** [comment]
https://issues.liferay.com/browse/COMMERCE-6087, cc: @PietroBocale

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 1]
```diff
@@ -66,12 +66,70 @@ definition {
...
+			channelCurrency = "US Dollar",
+			channelName = "Test Channel",
+			channelType = "Site",
+			siteName = "Guest");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		Refresh();
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:89]: I think it's a duplicate and can be removed

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 2]
```diff
@@ -66,12 +66,70 @@ definition {
...
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "ojma___toggle-switch",
+			locator1 = "ToggleSwitch#ENABLE_SPECIFIC_TOGGLE_SWITCH");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:104]: I have requested changes to the locator in this PR: https://github.com/liferay-commerce/liferay-portal/pull/903

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 3]
```diff
@@ -66,12 +66,70 @@ definition {
...
+
+		Click(locator1 = "Button#PLUS");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_BODY");
+
+		Check(
+			key_checkboxName = "Test Channel",
+			locator1 = "Checkbox#ENABLE_CHECKBOX_SELECT_CHANNEL");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:112]: I have requested changes to the locator in this PR: #903

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 4]
```diff
@@ -66,12 +66,70 @@ definition {
...
+
+		Button.click(button = "Delete");
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+
+
+
+		Click(
+			key_toggleSwitch = "ojma___toggle-switch",
+			locator1 = "ToggleSwitch#ENABLE_SPECIFIC_TOGGLE_SWITCH");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:128]: I have requested changes to the locator in this PR: #903

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path [hunk 5]
```diff
@@ -63,6 +63,13 @@
 	<td>//label[contains(.,'Use Custom Title')]//input[@type='checkbox']</td>
 	<td></td>
 </tr>
+
+<tr>
+    <td>ENABLE_SPECIFIC_TOGGLE_SWITCH</td>
+    <td>//span[contains(@id,'${key_toggleSwitch}')]</td>
+    <td></td>
+</tr>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path:71]: I have requested changes this path in this PR: #903

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path [hunk 6]
```diff
@@ -133,6 +133,13 @@
 	<td>//input[contains(@name,'displayImmediately')]</td>
 	<td></td>
 </tr>
+
+<tr>
+<td>ENABLE_CHECKBOX_SELECT_CHANNEL</td>
+    <td>//tr[contains(@data-name,'${key_checkboxName}')]//input[contains(@type,'checkbox')]</td>
+    <td></td>
+</tr>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path:141]: I have requested changes this path in this PR: #903

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Pietro69** [comment]
Please add the `property test.name.skip.portal.instance = "CPCommerceProductVisibility#RemoveAChannelFilter"`

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
Element is not present at "//tr/td[contains(.,'Guest')]//..//button[contains(.,'Choose')]"

**Comment by p-albuquerque** [comment]
Pleased close.

New PR sent in: #1232

**Pull request closed without merging**